Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -54,6 +55,7 @@ import ShadowMode from "../ShadowMode.js";
  * @param {Boolean} [options.backFaceCulling=true] Whether to cull back-facing geometry. When true, back face culling is determined by the material's doubleSided property; when false, back face culling is disabled. Back faces are not culled if the model's color is translucent.
  * @param {ShadowMode} [options.shadows=ShadowMode.ENABLED] Determines whether the model casts or receives shadows from light sources.
  * @param {Boolean} [options.showCreditsOnScreen=false] Whether to display the credits of this model on screen.
+ * @param {SplitDirection} [options.splitDirection=SplitDirection.NONE] The {@link SplitDirection} split to apply to this model.
```

Comment:
Assume we want to support individual glTF models or tilesets transcoded from the 1.0 `b3dm`/`i3dm`/`pnts`, you also need to update `fromGltf` 

Though as of yesterday, I had refactored the code in https://github.com/CesiumGS/cesium/pull/10214, so please merge `main` into this PR and update `makeModelOptions()` at the bottom of `ModelExperimental.js`

Reply:
By the same token, `ModelExperimental3DTileContent` needs similar updates to pass this parameter through.